CONTRIBUTING.md

```diff
@@ -2,45 +2,41 @@
 
 Thank you for taking the time to contribute. We appreciate it!
 
-There are two ways to contribute to this effort. The first way is to use this project's [Issues Page](https://github.com/ga4gh/ReadTaskTeam/issues), which we use as a forum to discuss both major and minor issues related to developing the GA4GH schemas and API definitions. Examples of the type of issues that can be submitted are:
+The GA4GH schemas define an API for sharing genomic and phenotypic data. Supporting the schemas is a [reference server](https://github.com/ga4gh/server), which acts as a useful, prototypical implementation of the API, and a [compliance suite](https://github.com/ga4gh/compliance), which provides data and functions to test an API implementation. Development of these three aspects of the project are coordinated, so that changes to the schemas are reflected in the creation of a working prototype within the reference server and compliance tests that establish conformance. Coordination in this way ensures the alliance is building software and standards that work for genomics.
```

Comment:
Can you split lines at ~80 chars please? Very long lines make diffs difficult to interpret.